packages/atomic/cypress/integration/result-list/result-table.cypress.ts

```diff
@@ -45,6 +45,7 @@ describe('Result List (Table) Component', () => {
     });
 
     it('should render labels in their corresponding header', () => {
+      ResultTableSelectors.firstRow().should('exist');
```

Comment:
Does that wait, or it will just fail ealier?

Reply:
The purpose of it was to wait, but it looks like that's not fixing the issue